helpful.el

```diff
@@ -747,6 +747,10 @@ For example, \"(some-func FOO &optional BAR)\"."
     (unless sym
       (user-error "No command is bound to %s"
                   (key-description key-sequence)))
+    (unless (commandp sym)
+      (user-error "%s is bound to symbol - %s, that is not a command"
```

Comment:
Why do you have a `-` in the middle of the sentence?

Reply:
@DamienCassou What would you like the sentence to be?

Reply:
I thought 
symbol <dash> symbol_name is better than 
symbol symbol_name

for some reason when I wrote this. Now that we are discussing this, I don't seem to be very confident about it.

Reply:
> What would you like the sentence to be?

I guess @Wilfred would be the best person to decide that, but I think your sentence without the dash and comma would be good enough.